website/content/en/blog/2024-09-18-platform-engineering-maturity-model-workshop.md

```diff
@@ -0,0 +1,66 @@
+---
+title:  'Accelerating Platform Engineering Value: Introducing PEMM Leveling Workshop'
+slug:   platform-engineering-maturity-model-workshop
+date:   2024-09-18 20:04:00 -0700
+author: Tyler Pate
+categories:
+- Article
+tags:
+- WG Platforms
+- Community Contribution
+---
+
+As organizations navigate the complexities of modern technology infrastructure, **platform engineering** has emerged as a critical discipline for driving efficiency and scalability. With the [**Platform Engineering Maturity Model (PEMM)**](https://tag-app-delivery.cncf.io/whitepapers/platform-eng-maturity-model/) being a relatively new concept, leveraging it effectively positions practitioners at the forefront of industry innovation. At the same time, our community continues to develop best practices for linking PEMM concepts to business value. Connecting our investment into PEMM methodology with clear business value is critical for buy-in with decision makers.
+
+In this case study, I'll demonstrate how I used the PEMM as tool to unify multiple teams following an internal restructuring at Red Hat. I'll demonstrate additional potential for the Platform Engineering Maturity Model (PEMM) to add value, connect teams, and align leadership with common terminology and goals.
+
+* [PEMM Leveling Workshop Template: Google Docs](https://docs.google.com/document/d/1h_ZoX89ScG0m10VxT5p7wwLlHMnVEv6RmrygvSLsnh4/edit?usp=sharing)
```

Comment:
We can remove this from here since we're already linking to the same document under the Highlights->Preparation section.